easybuild/easyconfigs/m/MATLAB/MATLAB-R2021a.eb

```diff
@@ -0,0 +1,33 @@
+# contributed by Luca Marsella (CSCS)
+name = 'MATLAB'
+version = 'R2021a'
+
+homepage = 'http://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = SYSTEM
+
+sources = ['/apps/common/UES/easybuild/sources/m/%s/%s.tar' % (name, version)]
+
+java_options = '-Xmx2048m'
+
+# read local license file
+import os as local_os
+local_licensefile = '/apps/common/UES/easybuild/sources/m/MATLAB/' + '%s-%s.txt' % (name, version)
+local_filehandle = local_os.open(local_licensefile, local_os.O_RDONLY)
+local_bytext = local_os.read(local_filehandle, 1000)
+local_text = local_bytext.decode('utf-8')
+# FIXME: the open() function raises the error below only with EasyBuild 4.4.0 and newer, not with 4.3.4
+# Failed to copy 'local_filehandle' easyconfig parameter: cannot serialize '_io.TextIOWrapper' object
+# with open(local_licensefile, 'r') as local_filehandle:
```

Comment:
Can you try the prior solution with the normal `open` and just set `local_filehandle` to `None` after the `with` block?

Reply:
Thanks for the advice @teojgo, it works: I will push the change in the pull request